src/integration/pages/idxPage/ukraine_in_russian/amp.test.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @service ukrainian
+ * @pathname /ukrainian/ukraine_in_russian
+ */
+
+import runAmpTests from '../ampTests';
+
+describe('AMP ukraine_in_russian IDX page', runAmpTests);
```

Comment:
Probably not necessary, but you could put the service name in the name of the test too
```suggestion
describe('AMP ukrainian/ukraine_in_russian IDX page', runAmpTests);
```